include/internal/catch_test_registry.h

```diff
@@ -52,6 +54,8 @@ struct AutoReg : NonCopyable {
 #define INTERNAL_CATCH_DEF(...) INTERNAL_CATCH_DEF __VA_ARGS__
 #define INTERNAL_CATCH_NOINTERNAL_CATCH_DEF
 
+#define INTERNAL_CATCH_EXPAND3(...) INTERNAL_CATCH_EXPAND1(INTERNAL_CATCH_DEF __VA_ARGS__)
```

Comment:
I would prefer more descriptive name for this, e.g. `INTERNAL_CATCH_REMOVE_PARENS`